tests/integration/test_decompositions_integration.py

```diff
@@ -590,21 +590,20 @@ def test_CZgate_decomp_equal(self, setup_eng, s, tol):
         eng.run(prog)
         assert np.all(eng.backend.is_vacuum(tol))
 
-    @pytest.mark.parametrize('r', np.linspace(0., 0.6, 5))
+    @pytest.mark.parametrize('r', np.linspace(-0.3, 0.3, 5))
```

Comment:
Does this one have the same problem as the FIXMEs above?

Reply:
No, this one's fine. I just changed the testset so that it also includes negative values.